_posts/2020-04-21-week5-winner.md

```diff
@@ -0,0 +1,15 @@
+---
+author: John
+title: Congratulations Week Five Winner: Open Marketplace!
+---
+
+Our week five winner [Open Marketplace](https://open-marketplace-applications.github.io/marketplace/), is leveraging the open source and nonprofit model used by Wikipedia to improve Covid-19 response.
+
+<img src="/images/blog/open marketplace.jpg" alt="Get Cleard" style="max-width: 100%;">
```

Comment:
Can you change the alt text here to Open Marketplace?

Reply:
dough!  cut and paste can be so dangerous